addons/web/static/src/core/domain_selector/domain_selector_fields.js

```diff
@@ -236,7 +236,32 @@ const SELECTION = {
         in: SELECTION_EDITOR_IN,
         not_in: SELECTION_EDITOR_IN,
     },
-    defaultValue: (field) => field.selection[0][0],
+    defaultValue: (field) => field.selection[0][0] ?? false,
```

Comment:
why not `field.selection?.[0]?.[0] ?? false` like below

Reply:
I suppose that selection fields always have at least one option
but I know that selection properties could not have options